docs/security/trust/content_trust.md

```diff
@@ -0,0 +1,261 @@
+<!--[metadata]>
++++
+title = "Content trust in Docker"
+description = "Enabling content trust in Docker"
+keywords = ["trust, security, docker,  documentation"]
+[menu.main]
+parent= "smn_content_trust"
+weight=-1
++++
+<![end-metadata]-->
+
+# Content trust in Docker
+
+When transferring data among networked systems, *trust* is a central concern.
+When using Docker you push and pull images (data) to a registry.  Currently,
+trust is supported for users of Docker's public Hub. It is not available for
+Docker Trusted Registry or for private registries. 
+
+## Understand trust in Docker 
+
+Content trust allows operations with a remote Docker registry to enforce
+client-side signing and verification of image tags. Currently, content trust is
+disabled by default. You must enabled it by setting the `DOCKER_CONTENT_TRUST`
+variable.
+
+Once trust is enabled, image creators can sign their images. Image consumers can
+ensure that the images they use are signed. Creators and consumers can be
+individuals alone or in organizations. Docker's trust feature supports users and
+automated processes such as builds. 
+
+### Image tags and trust
+
+An individual image record has the following identifier:
+
+```
+[REGISTRY_HOST[:REGISTRY_PORT]/]REPOSITORY[:TAG]
+```
+
+A particular image `REPOSITORY` can have multiple tags. For example, `latest` and
+ `3.1.2` are both tags on the `mongo` image. An image creator can build a image
+ and tag combination many times changing the image with each build. 
+
+Trust applies to the `TAG` portion of an image. Each image repository has a set
+of keys that image creators use to sign an image tag. Image creators have
+discretion on which tags they sign.
+
+An image repository can contain an image with one tag that is signed and another
+tag that is not. For example, consider [the Mongo image
+repository](https://registry.hub.docker.com/_/mongo/tags/manage/). The `latest`
+tag could be unsigned while the `3.1.6` tag could be signed. It is the
+responsibility of the image creator to determine if an image tag is signed or
+not. In this representation, some image tags are signed, others are not:
+
+![Signed tags](../images/tag_signing.png)
+
+Currently, it is also possible to sign some image tag pushes and not others. For
```

Comment:
What about something like: `Publishers have the choice of whether to sign a specific tag or not. It is  possible for the content of an unsigned tag to not match the content of a signed tag with the same name. In fact, this allows publishers to iterate over the unsigned version of an image before officially signing it. For example, a publisher can push a tagged image `someimage:latest` and sign it. Later the same publisher can push and not sign `someimage:latest`, and while this push will replace the unsigned tag `latest` it will not affect the signed version of `latest``